CHANGES.md

```diff
@@ -1,3 +1,17 @@
+## edge-19.9.3
+
+* Helm
+  * Allow disabling namespace creation during install (thanks @KIVagant!)
+* Controller
+  * Fixed proxy injector timeout during a large number of concurrent injections
+  * Separated the destination controller into its own separate pod
+  * Updated Prometheus config to keep only needed cadvisor metrics
+* Web UI
+  * Fixed bad request in the top routes tab on empty fields (thanks @pierDipi!)
+* Proxy
+  * Fixes to the client's backoff logic
+  * Add 587 to the list of ports to disable protocol detection
```

Comment:
might be nice to call out why.  something to do with smtp?

Reply:
Yeah, just another common port for SMPT.  @adleong does this look fine for you?
```
Added 587 (SMTP) to the list of ports disabled by default (bound to server-speaks-first protocols)
```

Reply:
:+1: